excellence-fee/order_admin/admin_table_value.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+
+	global $post;
+    global $wpdb;
+
+    if ( 'excellence_fee' === $column ) {
+
+        // $order    = wc_get_order( $post->ID );
+        
+        $existing_order = $wpdb->get_results('select order_item_id from ' . $wpdb->prefix . 'woocommerce_order_items where order_id="'.$post->ID.'" and order_item_name="Excellence Fee"');
+			if($existing_order){
+				foreach ($existing_order as $key => $value) {
+			            $existing_value_id=$value -> order_item_id;
+			            
+			        }
+
+			        $order1    = wc_get_order_item_meta( $existing_value_id, $key, $single );
```

Comment:
@Shraddha1596 wrong naming convention for variable, what's order1 like variable ????????